ipc/arwenpart/messenger.go

```diff
@@ -4,19 +4,21 @@ import (
 	"os"
 
 	"github.com/ElrondNetwork/arwen-wasm-vm/ipc/common"
-	"github.com/ElrondNetwork/arwen-wasm-vm/ipc/logger"
 	"github.com/ElrondNetwork/arwen-wasm-vm/ipc/marshaling"
+	logger "github.com/ElrondNetwork/elrond-go-logger"
 )
 
+var messengerLog = logger.GetOrCreate("arwen/messenger")
```

Comment:
Can `messengerLog` be renamed to `log`, like in the other files?

Reply:
I wanted to have separate loggers for `arwen/part` and `arwen/messenger`.

Reply:
Fixed :)